source/collections.txt

```diff
@@ -10,6 +10,9 @@ Collections
    :name: genre
    :values: tutorial
 
+.. meta:: 
+   :description: How to create or drop a collection, which is a grouping of MongoDB documents equivalent to a table in a relational database system.
```

Comment:
(suggestion)
Consider dropping the phrase explaining collections. So just something like: 
"How to use Compass to create or drop a MongoDB collection." 

My reasoning is that the collection definition might be too much detail for a summary page.

Reply:
Ian mentioned putting 'table' in and said there wasn't information anywhere that told the user that tables are collections for us.  I thought that was a good idea to make sure that we've got that additional way to access it.  What do you think?